app/assets/javascripts/leaflet.map.js

```diff
@@ -369,6 +368,14 @@ L.extend(L.Icon.Default.prototype, {
   }
 });
 
+OSM.boundsArea = function ([[minLat, minLon], [maxLat, maxLon]]) {
+  const height = maxLat - minLat;
+  let width = maxLon - minLon;
+
+  if (width < 0) width += 360;
```

Comment:
This fix is specific to what Overpass returns. You want to put it into a generic area calculation. Do you think it's going to work for all possible bounding boxes?